### ISSUE TITLE
nn.Linear 

### ISSUE THREAD

**Issue by daredevil6776** [WuJie1010/Facial-Expression-Recognition.Pytorch#80]
How can we assure that the predicted probabilities of all the classes sums to one using nn.Linear?

**Comment by WuJie1010** [comment]
softmax helps to finish it

**Comment by daredevil6776** [comment]
> softmax helps to finish it
But in training or testing there is no softmax layer specified

**Comment by WuJie1010** [comment]
> > softmax helps to finish it
> > But in training or testing there is no softmax layer specified

softmax operation is in the cross-entropy loss function
